lib/controllers/project.js

```diff
@@ -106,6 +106,17 @@ Project.prototype.status = function(projectId, callback) {
 };
 
 //-----------------------------------------------------------------------------
+Project.prototype.resize = function(projectId, servoSize, callback) {
+  librarian.project.update( { id: projectId, servoSize: servoSize }, userConfig.data.apiKey, function(err, status) {
+    if (err) {
+      return callback(err, null);
```

Comment:
On line 112? It's a matter of clarity. Use lines 75, 101, and 291 as other instances which I did not write.